xrpl/models/transactions/types/transaction_type.py

```diff
@@ -18,6 +18,7 @@ class TransactionType(str, Enum):
     CHECK_CASH = "CheckCash"
     CHECK_CREATE = "CheckCreate"
     CLAWBACK = "Clawback"
+    DELETE_ORACLE = "OracleDelete"
```

Comment:
These should match the actual transaction type name, not what they are in rippled - so `ORACLE_DELETE` and `ORACLE_SET`.

Reply:
fixed in [df462ba](https://github.com/XRPLF/xrpl-py/pull/701/commits/df462ba5ca42b12e4ab754c9c6446b990264f6da)